sound/soc/intel/boards/sof_rt5682.c

```diff
@@ -619,7 +649,7 @@ static int sof_audio_probe(struct platform_device *pdev)
 	ssp_codec = sof_rt5682_quirk & SOF_RT5682_SSP_CODEC_MASK;
 
 	/* compute number of dai links */
-	sof_audio_card_rt5682.num_links = 1 + dmic_be_num + hdmi_num;
+	sof_audio_card_rt5682.num_links = 1 + dmic_be_num + hdmi_num + 2;
```

Comment:
please add comment to say that the +2 is for probe playback/capture.

Reply:
Hmm, there is non for SSP? (+1)?  Can add that anyway, but there is some inconsistency here.

Reply:
yeah there's always inconsistencies with existing code. Its OK when reading this patch but when someone tries to reach the code after it is merged, the +1 and +2 could take a while to figure out.

Reply:
Ack.